src/player.js

```diff
@@ -2120,6 +2120,18 @@ export default class Player extends FakeEventTarget {
       }
     });
 
+    const isMuted = () => {
+      Player._logger.debug('Checking muted value');
+      // at this point it is possible that the engine hasn't loaded yet, so this.muted is not reflecting the actual state.
+      // first, check if someone set a value in muted; if not, then check the config; lastly, check the engine.
+      const muted =
+        (typeof this._playbackAttributesState.muted === 'boolean' && this._playbackAttributesState.muted) ||
+        this._config.playback.muted ||
```

Comment:
this._config.playback.muted can also be undefined, and we want to distinguish between undefined and false